src/lib/core/components/BirdsEyeView.tsx

```diff
@@ -25,19 +27,17 @@ export function BirdsEyeView(props: Props) {
     completeCasesAxesVars,
     stratificationIsActive,
     enableSpinner = false,
+    totalCounts,
```

Comment:
I think we can lose the `filters` prop now.  Will review and test properly later.  Just had a quick look!